sa.tf

```
@@ -67,6 +67,24 @@ module "final_storage_account" {
   common_tags = var.common_tags
 }
 
+module "streaming_storage_account" {
+  source                   = "git@github.com:hmcts/cnp-module-storage-account?ref=master"
+  env                      = var.env
+  storage_account_name     = replace("${var.product}streamingsa${var.env}", "-", "")
```

avoid adding type to global resources less likely to hit length limits
```suggestion
  storage_account_name     = replace("${var.product}streaming${var.env}", "-", "")
```

---

modified